rolandd_hw03.py

```diff
@@ -0,0 +1,188 @@
+
+
+#################################################################################
+# Author: Dylan Roland
+# Username: rolandd
+#
+# Assignment: HW03
+# Purpose: To draw a giraffe
+# Google Doc Link:https://docs.google.com/document/d/1A-STS7gX_5MVm1CM2mSFn-NCOdlrcuB4ouTSuaZ2fhc/edit
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+giraffe = turtle.Turtle()
+wn=turtle.Screen()
+wn.bgcolor('green')
+giraffe.speed(0.1)
```

Comment:
NO code outside of main() or function definitions now!

These four lines belong as the first four lines of your `main()` function